src/Helper/ExportHelper.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php /** @noinspection ALL */
```

Comment:
Avoid this, please only suppress what you need

Reply:
@supercid I can't find the proper inspection tag for the "Qualifier can be replaced with an import" error.
Referring to this list of tags: https://gist.github.com/gskema/66f82792f2221771ea7407ff5245b35a
Also my IDE doesn't throw this error.

Reply:
@supercid @olsi-qose tried to suppress specific errors, for some reason it is not working. Not sure if it's worth spending more time on this. What if we just take a note to remove all suppressions once we can remove the phpseclib/phpseclib:2.x.x altogether since this whole workaround is a bit hack-ish?